pom.xml

```diff
@@ -67,7 +67,7 @@
         <dependency>
             <groupId>org.liquibase</groupId>
             <artifactId>liquibase-core</artifactId>
-            <version>4.23.2</version>
+            <version>0-SNAPSHOT</version>
```

Comment:
I don't know who owns this repo, but I'm betting that this is not a change that is okay to make. Can you check with whoever owns this?

Reply:
Usually it's me ...